scripts/rollup/build.js

```diff
@@ -584,7 +588,11 @@ function handleRollupWarning(warning) {
     return;
   }
 
-  if (typeof warning.code === 'string') {
+  const ignoredRollupWarnings = ['CIRCULAR_DEPENDENCY'];
+
+  if (ignoredRollupWarnings.indexOf(warning.code) !== -1) {
```

Comment:
Maybe simplify like this `warning.code === 'CIRCULAR_DEPENDENCY'`?

Reply:
I could do that.

The intend was to make it extensible where we could put more items to ignore if needed.